src/graphql/admin/lexicographic-sort-schema.graphql

```diff
@@ -1,8 +1,17 @@
-interface Account {
-  csvTransactions(walletIds: [WalletId!]!): String!
-  defaultWalletId: WalletId!
+type Account {
```

Comment:
oh could it have anything to do with a mix between `interface` and `type` that I may have done?

Reply:
@samerbuna

Reply:
Don't understand this comment.